lib/command.c

```diff
@@ -263,8 +266,10 @@ void install_node(struct cmd_node *node)
 		cmd_token_new(START_TKN, CMD_ATTR_NORMAL, NULL, NULL);
 	graph_new_node(node->cmdgraph, token,
 		       (void (*)(void *)) & cmd_token_del);
-	node->cmd_hash = hash_create_size(16, cmd_hash_key, cmd_hash_cmp,
-					  "Command Hash");
+
+	snprintf(hash_name, CMD_HASH_STR_SIZE, "Command Hash: %s", node->name);
```

Comment:
`sizeof(hash_name)` please so that this can't be accidentally changed later

Reply:
fixed